Release/include/cpprest/details/http_client_impl.h

```diff
@@ -296,9 +296,10 @@ class _http_client_communicator
     // URI to connect to.
     const http::uri m_uri;
 
-private:
 
-    http_client_config m_client_config;
+	http_client_config m_client_config;
```

Comment:
exposing non-`const` access to this data member could be dangerous. The one new use in http_client_asio.cpp can call the member function client_config() for this.